pyproject.toml

```diff
@@ -2,9 +2,12 @@
 requires = ["setuptools>=61.0", "wheel"]
 build-backend = "setuptools.build_meta"
 
+[global-variables]
+mute_period = 0
```

Comment:
I think it's cleaner to create a separate config file for application logic, since pyproject.toml is only meant for build configuration, packaging and tools